unit_tests/test_scan_operation_thread.py

```diff
@@ -15,7 +15,17 @@
 
 # from sdcm.utils.operations_thread import ThreadParams
 from unit_tests.test_cluster import DummyDbCluster, DummyNode
-from sdcm.scan_operation_thread import ScanOperationThread, ThreadParams, PrometheusDBStats
+from sdcm.utils.decorators import retrying, Retry
+
+
+def mock_retrying_decorator(*args, **kwargs):  # pylint: disable=unused-argument
+    """Decorate by doing nothing."""
+    return retrying(1, 1, allowed_exceptions=(Retry, ))
+
+
+patch('sdcm.utils.decorators.retrying', mock_retrying_decorator).start()
```

Comment:
Please, don't do it this way.
Patch stuff locally either by using context manager way (`with ...`) or patching test methods...

Reply:
with patch "retrying" 
======================== 1 passed, 8 warnings in 11.10s ========================
without 

=================== 1 passed, 8 warnings in 71.16s (0:01:11) ===================

Reply:
but I see what you mean - "it is not safe to do like this" .

Reply:
@vponomaryov  Updated

it is still globally but now it should be safe and works only within one file